conda/recipes/cuml/meta.yaml

```diff
@@ -33,40 +33,54 @@ build:
     - SCCACHE_S3_KEY_PREFIX=cuml-linux64 # [linux64]
     - SCCACHE_S3_USE_SSL
   ignore_run_exports_from:
-    - {{ compiler('cuda') }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }}
+    {% endif %}
 
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} ={{ cuda_version }}
+    {% else %}
+    - {{ compiler('cuda') }}
+    {% endif %}
+    - cuda-version ={{ cuda_version }}
     - cmake {{ cmake_version }}
     - ninja
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
-    - cuda-python >=11.7.1,<12.0
-    - cudatoolkit ={{ cuda_version }}
+    - cuda-version ={{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - cuda-python ==11.7.1
+    - cudatoolkit
+    - libcublas {{ cuda11_libcublas_host_version }}
+    - libcurand {{ cuda11_libcurand_host_version }}
+    - libcusolver {{ cuda11_libcusolver_host_version }}
+    - libcusparse {{ cuda11_libcusparse_host_version }}
+    {% else %}
+    - cuda-python ==12.0.0
+    - libcublas
```

Comment:
Is libcufft missing? (Or can all these math libraries be removed from the Python packaging, see other comment?)
```suggestion
    - libcublas
    - libcufft
```